defaultSystem/geode.properties

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Default Gemfire C++ distributed system properties
+# Default C++ distributed system properties
```

Comment:
These properties are also loaded by the .NET client. I wonder if we can give them a better name.

Reply:
Geode Native probably. This file shouldn't get confused with the one for the Java client/server.